android/src/main/kotlin/dev/rexios/polar/PolarPlugin.kt

```diff
@@ -105,18 +94,13 @@ class PolarPlugin :
         searchChannel = EventChannel(flutterPluginBinding.binaryMessenger, "polar/search")
         searchChannel.setStreamHandler(searchHandler)
 
-        if (wrapperInternal == null) {
```

Comment:
Had issue with this when attempting to use workmanager to run background tasks on android. Since it loads the flutter engine in a new isolate this onAttachedToEngine will run for every plugin and it would kill/takeover the polar SDK killing communication from the main app. Even after, re-initializing would not work (it would then only communicate battery life 🤷 ).

Changes make initialization and shutdown their own callable functions - however I wouldn't recommend anyone ever calling `shutdown` since its not recoverable without a complete restart (and its Android only). The above issue did not exist on iOS but the handler on that has been changed to match this pattern.